### ISSUE TITLE
GUI stuck trying to create PlotNFT [Bug] 

### ISSUE THREAD

**Issue by vernzimm** [Chia-Network/chia-blockchain#9970]
### What happened?

This may have happened because of the corruption I experienced in issue #9677 but I'm not sure. When I was trying to fix it, I eventually deleted main db, uninstalled, renamed .chia folder, then installed again. I moved my wallet back into the new .chia folder.

I didn't see any PlotNFTs at first (I'm a little hazy on how this happened, sorry). I tried to create a new one (no 2), and somehow tried to create another new one (no 3), but I was not synced, so they were both on hold.

Once I finally synced up again, my original (no 1) reappeared with all the plots assigned to it, great. (no 2) also processed once the transactions could go through and was available, but I didn't need it, so I changed it to self pool (since we can't delete at the moment).

(no 3) on the other hand, has continued to be in the "Waiting for the transaction to be confirmed" state with rolling circle in GUI. Somehow the transaction for it got lost? I can't cancel it, and it can't complete the creation. When I click on the "Pool" tab in GUI, the log is spammed every second (included in log output)...

I have heard about removing entries in config that would hide them in GUI, so I checked what was in there. Only the original (no 1) and the self pool (no 2) are listed there (also included in log output).

One more thing to note, (no 3) is attempting to join chia.futurepool.io, same as (no 1) and (no 2). I don't think there is any authentication that happens at the pool host though, so I'm assuming it's unrelated.

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
2022-01-26T13:10:43.101 wallet chia.rpc.rpc_server        : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\rpc_server.py", line 217, in safe_handle
  File "chia\rpc\rpc_server.py", line 208, in ws_api
  File "chia\rpc\wallet_rpc_api.py", line 652, in get_transaction
ValueError: Transaction 0xaaa... not found


......... Config.yaml "Pool" section .........
pool:
  logging: *id001
  network_overrides: *id002
  pool_list:
  - authentication_public_key: '0xb3d...'
    launcher_id: '0x139...'
    owner_public_key: '0xab2...'
    p2_singleton_puzzle_hash: '0xe3f...'
    payout_instructions: bf84b...
    pool_url: ''
    target_puzzle_hash: '0x2c9...'
  - authentication_public_key: '0x913..'
    launcher_id: '0x739...'
    owner_public_key: '0xa5a...'
    p2_singleton_puzzle_hash: '0xdec...'
    payout_instructions: 58eff...
    pool_url: https://chia.futurepool.io
    target_puzzle_hash: '0x5a2...'
  selected_network: mainnet
  xch_target_address: xch1l...
```

**Comment by vernzimm** [comment]
![Pool Stuck](https://user-images.githubusercontent.com/29264909/151418818-126626c3-db8d-4f74-842f-759e60f2dad4.jpg)

**Comment by danarbraz** [comment]
I have the same issue (same symptoms at least) with 1.2.12-dev354. 

2022-02-22T20:30:00.832 wallet chia.rpc.rpc_server        : WARNING  Error while handling message: Traceback (most recent call last):
  File "/home/chia/chia-blockchain/chia/rpc/rpc_server.py", line 215, in safe_handle
    response = await self.ws_api(message)
  File "/home/chia/chia-blockchain/chia/rpc/rpc_server.py", line 206, in ws_api
    return await f(data)
  File "/home/chia/chia-blockchain/chia/rpc/wallet_rpc_api.py", line 666, in get_transaction
    raise ValueError(f"Transaction 0x{transaction_id.hex()} not found")
ValueError: Transaction 0xa.... not found

**Comment by darwinjobchia** [comment]
I have the same too

**Comment by darwinjobchia** [comment]
![Screenshot 2022-02-25 194019](https://user-images.githubusercontent.com/83147673/155701310-9268ffed-b1b5-4a94-99f3-ff77014b4695.jpg)

**Comment by darwinjobchia** [comment]
it is rolling forever

**Comment by vernzimm** [comment]
This appears to have been resolved successfully with 1.3.0.
